src/components/Button/Button.style.ts

```diff
@@ -338,8 +359,14 @@ function buttonVariants({
         }
       `;
     default:
+      const defaultActiveCSS = css`
+        background: ${activeColor};
+        transform: scale(1);
```

Comment:
This should be shared with the default active psuedoclass. Also, the current default active psuedoclass has a different scale so those need to be aligned.

Reply:
done and aligned without scale